framework/src/task_manager/task_manager_core.c

```diff
@@ -216,8 +216,12 @@ static int taskmgr_unregister(int handle)
 #endif
 
 	TM_PID(handle) = 0;
-	TM_STOP_CB(handle) = NULL;
-	TM_EXIT_CB(handle) = NULL;
+	if (TM_STOP_CB_INFO(handle) != NULL) {
+		TM_FREE(TM_STOP_CB_INFO(handle));
```

Comment:
I think TM_STOP_CB_INFO(handle) = NULL; is needed.

Reply:
Done